core/templates/dev/head/pages/exploration_editor/translation_tab/StateTranslationDirective.js

```diff
@@ -261,15 +268,7 @@ oppia.directive('stateTranslation', [
           });
 
           $scope.initStateTranslation = function() {
-            $scope.activatedTabId = $scope.TAB_ID_CONTENT;
```

Comment:
Why remove this?

Reply:
This is being already activated [here](https://github.com/oppia/oppia/pull/6211/files#diff-7c248bde202b50468bd3ebce03436662R57), also while refresh we will activate the content tab once the content data is loaded into the scope over [here](https://github.com/oppia/oppia/pull/6211/files#diff-7c248bde202b50468bd3ebce03436662R291).

Reply:
Ah ok. Thanks!